django/getting-started/index.html.md

```diff
@@ -16,156 +16,233 @@ related_pages:
 ]
 ```
 
-And update the existing `demo/urls.py` file as well for project-level URL configuration.
+And update the existing [`hello_django/urls.py`](https://github.com/fly-apps/hello-django/blob/main/hello_django/urls.py) file for project-level URL configuration:
 
 ```python
-# demo/urls.py
+# hello_django/urls.py
 from django.contrib import admin
-from django.urls import path, include  # new
+from django.urls import include, path
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
-    path("", include("fly.urls")),  # new
+    path('admin/', admin.site.urls),
+    path('', include('hello.urls'))  # <-- Updated!
 ]
 ```
 
-That's it! Run the `migrate` command to initialize our local database.
+### Run migrations
+
+As part of the Django's core functionality, some existing apps are included by default to provide you with out-of-the-box features. Some of those apps require their own database tables.
+
+To initialize the local database and set up those tables, run the `migrate` command:
 
 ```cmd
 python manage.py migrate
 ```
 
-Now `runserver` to start up Django's local web server.
+### Start the Web Server
+
+Now `runserver` to start up Django's local web server:
 
 ```cmd
 python manage.py runserver
 ```
 
-If you open `http://127.0.0.1:8000/` in your web browser it now displays the text "Hello, Fly!"
+If you open `http://127.0.0.1:8000/` in your web browser it now displays the text `Hello, Fly!`.
 
 ## Django Deployment Checklist
 
-By default, Django is configured for local development. The [How to Deploy Django](https://docs.djangoproject.com/en/stable/howto/deployment/) and [Django deployment checklist](https://docs.djangoproject.com/en/stable/howto/deployment/checklist/) guide list the various steps required for a secure deployment. However, for demonstration purposes, we can take some shortcuts.
+By default, Django is configured for local development. The [How to Deploy Django](https://docs.djangoproject.com/en/stable/howto/deployment/) and [Django Deployment Checklist](https://docs.djangoproject.com/en/stable/howto/deployment/checklist/) guide list the various steps required for a secure deployment.
+
+> You can also find a complete guide [Deploying Django to Production](https://fly.io/django-beats/deploying-django-to-production/) in our [Django Beats Blog](https://fly.io/django-beats/).
 
-First, in the `demo/settings.py` file update the `ALLOWED_HOSTS` configuration to accept all hosts.
+However, for demonstration purposes, we can take some shortcuts.
+
+First, in the [`hello_django/settings.py`](https://github.com/fly-apps/hello-django/blob/main/hello_django/settings.py) file update the `ALLOWED_HOSTS` configuration to accept all hosts:
 
 ```python
-# demo/settings.py
-ALLOWED_HOSTS = ["*"]  # new
+# hello_django/settings.py
+ALLOWED_HOSTS = ["*"]  # <-- Updated!
 ```
 
-Second, install [Gunicorn](https://gunicorn.org/) as our production server.
+Second, install [Gunicorn](https://gunicorn.org/) as our production server:
```

Comment:
```suggestion
Second, install [Gunicorn](https://gunicorn.org/) as our [production server](https://docs.djangoproject.com/en/dev/ref/django-admin/#runserver):
```
I was going to say that a few words explaining why runserver isn't a prod server would help people understand, but the link has a very good explanation.